File: app/src/main/java/com/bridge/androidtechnicaltest/db/AppDatabase.java
package com.bridge.androidtechnicaltest.db;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {
        Pupil.class
}, version = 1, exportSchema = false)

public abstract class AppDatabase extends RoomDatabase {
    public abstract PupilDao getPupilDao();
}

File: app/src/main/java/com/bridge/androidtechnicaltest/db/AppDatabase.kt
package com.bridge.androidtechnicaltest.db

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [Pupil::class, Cache::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract val pupilDao: PupilDao
}